sky/utils/cli_utils/status_utils.py

```diff
@@ -70,25 +74,19 @@ def show_status_table(cluster_records: List[Dict[str, Any]],
         pending_autostop += _is_pending_autostop(record)
 
     if cluster_records:
-        name = 'Clusters'
-        if is_reserved:
-            click.echo()
-            name = 'Reserved Clusters'
-        click.echo(f'{colorama.Fore.CYAN}{colorama.Style.BRIGHT}{name}: '
-                   f'{colorama.Style.RESET_ALL}')
-        click.echo(cluster_table)
-        if is_reserved:
+        if reserved_group_name is not None:
             click.echo(
-                f'{colorama.Style.DIM}Reserved clusters will be autostopped '
-                'when inactive. Refresh statuses with: sky status --refresh'
+                f'\n{colorama.Fore.CYAN}{colorama.Style.BRIGHT}'
+                f'{reserved_group_name}: {colorama.Style.RESET_ALL}'
+                f'{colorama.Style.DIM}(will be autostopped when inactive)'
```

Comment:
nice! "when inactive -> if idle for 30min"

Reply:
Done. Thanks!